lib/config/validation.ts

```diff
@@ -673,6 +677,29 @@ export async function validateConfig(
         }
       }
     }
+
+    if (key === 'hostRules' && is.array(val)) {
+      const allowedHeader = GlobalConfig.get('allowedHeader');
+      for (const rule of val as HostRule[]) {
+        if (!rule.header) {
+          continue;
+        }
+        for (const [header, value] of Object.entries(rule.header)) {
+          if (!is.string(value)) {
+            errors.push({
+              topic: 'Configuration Error',
+              message: `Invalid hostRules header value configuration: should be a string.`,
```

Comment:
```suggestion
              message: `Invalid hostRules header value configuration: header must be a string.`,
```

Fixing the same problem as above with this message.